### DISCUSSION TITLE
Hide start menu and clock on secondary screens

### DISCUSSION THREAD

**Discussion by ygoe** [ramensoftware/windhawk-mods#305] [General]
Hello,

Could I use the Windows 11 Taskbar Styler mod to hide the start menu button and clock area on secondary screens? I've set up my taskbar to only show windows on the current screen for additional screens, and the start menu and clock are kind of redundant there. I'm not sure whether I'll actually like hiding those elements, but I'd like to try it out.

**Comment by m417z** [comment]
Yes, Windows 11 Taskbar Styler affects all taskbars.

![image](https://github.com/ramensoftware/windhawk-mods/assets/4129781/0eb5c2d5-d448-446c-b791-1be6e9a7ea82)

**Comment by TAHayden** [comment]
Like the OP I want to hide both the clock AND the 'Start' button on secondary monitors. From your screenshot, above, @m417z, I've managed to hide the clock but I can't see/derive the 'target' 'address' of the 'Start' button. I downloaded UWPSpy, but can't get a tree view like you had there so I'm struggling to suss this out myself.. Please can you help?! 😇
All the best,
-Tom

**Reply by m417z** [reply]
Hi Tom, currently the mod doesn't provide a way to distinguish the start button on secondary taskbars, as it doesn't have unique properties which can be used to match it specifically.

**Reply by TAHayden** [reply]
Thanks for your swift reply. 
Ahh, shame - that would really tidy things up when using the 'Dock' style with auto-hide and the clock hidden...
![2025-05-28 10_54_49-](https://github.com/user-attachments/assets/0dbc2da0-7b88-4389-aa32-7aa7ea75a76f)

Any chance another mod that specifically did this could be made or is it technically not possible?
